.github/workflows/development.yml

```diff
@@ -24,8 +24,8 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
-      - name: "🔧 install npm@7"
-        run: npm i -g npm@7
+      - name: "🔧 install npm@latest"
+        run: npm i -g npm@latest
```

Comment:
bold, but I think this development is a great place to test this out.

Reply:
the big issue with most people was upgrading 6->7, from here on I don't think it's an issue if we keep to the latest and catch bugs - however, am planning to abstract the workflows and make them reusable from open-sauced/open-sauced